### PR TITLE
Doc: Add completor Vim plugin

### PR DESCRIPTION
**Pull request by polyzen**

### CODE REVIEW

**Comment by adrienverge** [comment]
Thanks for contributing!

Syntastic and Completor look like to different things to me (a background checker and a typeahead completor), so can you rephrase a bit (especially "If you instead use") before I merge?

**Comment by polyzen** [comment]
:facepalm:.. Got so caught up with deciding how to add the information in, that I penned in the wrong plugin! Fixing.
